nntrainer/layers/flatten_layer.cpp

```diff
@@ -39,7 +60,8 @@ void FlattenLayer::finalize(InitLayerContext &context) {
 }
 
 void FlattenLayer::setProperty(const std::vector<std::string> &values) {
-  auto remain_props = loadProperties(values, reshape_props);
+  auto remain_props = loadProperties(values, flatten_props);
+  remain_props = loadProperties(remain_props, reshape_props);
```

Comment:
Uh... overwriting results into the same variable? Why? The first call becomes useless...

Reply:
Removed this line

Reply:
You didn't remove it....

Reply:
I'm sorry I was confused with @DonghakPark 's comment.
The function "loadProperties" loads only those properties from the first given list of properties that correspond to the second given argument list, and returns any non-matching arguments.
So first call is needed to load the "flatten properties" from the "values".